services/motion/builtin/builtin.go

```diff
@@ -89,7 +93,7 @@ func (ms *builtIn) Reconfigure(
 	ctx context.Context,
 	deps resource.Dependencies,
 	conf resource.Config,
-) (err error) {
```

Comment:
Named return variables are a sharp edge in go which I don't think we should be using unless there is a good reason to: https://go.dev/tour/basics/7